container-search/src/main/java/com/yahoo/search/searchchain/Execution.java

```diff
@@ -140,7 +140,7 @@ public Context(SearchChainRegistry searchChainRegistry, IndexFacts indexFacts, S
         public Context(SearchChainRegistry searchChainRegistry, IndexFacts indexFacts,
                        SpecialTokenRegistry tokenRegistry, RendererRegistry rendererRegistry, Linguistics linguistics,
                        Executor executor) {
-            this(searchChainRegistry, indexFacts, SchemaInfo.empty(), tokenRegistry, rendererRegistry, linguistics, Runnable::run);
+            this(searchChainRegistry, indexFacts, SchemaInfo.empty(), tokenRegistry, rendererRegistry, linguistics, executor);
```

Comment:
this change looks totally unrelated

Reply:
It fixes a bug in an unused method preserved for compatibility, introduced in an earlier part of this work.